Dockerfile

```diff
@@ -1,14 +1,18 @@
-FROM apache/airflow:2.6.0-python3.10
+FROM python:3.10
 
-ENV PYTHONBUFFERED=0 
+WORKDIR /opt/airflow/
+ENV AIRFLOW_HOME=/opt/airflow/
+
+ENV PYTHONBUFFERED=0
 ENV AIRFLOW_UID=501
 
-COPY requirements.txt ./requirements.txt
-COPY requirements-test.txt ./requirements-test.txt
+COPY pyproject.toml ./pyproject.toml
+COPY poetry.lock ./poetry.lock
+
 COPY dags ./dags
-USER airflow
 RUN pip install --upgrade pip &&\
     pip install --no-cache-dir --upgrade setuptools==59.1.1 &&\
     pip install --no-cache-dir --upgrade wheel &&\
-    pip install --no-cache-dir --user -r requirements.txt -r requirements-test.txt
-    
+    pip install --no-cache-dir poetry
+
+RUN poetry install
```

Comment:
hmm why we need to change completely the image?

Reply:
But we want to use poetry but not pip anymore. Instead of it, we had pip before:
`pip install --no-cache-dir --user -r requirements.txt -r requirements-test.txt`

Reply:
@ErnestaP i'm referring to the docker image that we are using here, it's changed from

```
FROM apache/airflow:2.6.0-python3.10
```

to 


```
FROM python:3.10
```

Is it any particular reason?

Reply:
@ErnestaP ☝️

Reply:
Sorry, I replied in the wrong place... 

Dockerfile - yes. The reason is I want to see if airflow runs by using poetry. That's why I am taking the Python image, installing poetry, and then running tests by using this container. 

https://github.com/cern-sis/workflows/blob/9a0124e82d48c505f910783547daa7ec6dc8532c/.github/workflows/test-and-build.yml#L72-L76 (the file is from Poetry PR)